### DISCUSSION TITLE
Image scene preview in spoke

### DISCUSSION THREAD

**Discussion by salahzar** [mozilla/hubs#3020] [General Questions & Discussion] [answered]
I thought spoke was using the current cam as preview image for the scene, but now I tried and was not able to set it up.
I read one question suggesting to use firefox instead of chrome, but this did not work. Is there a procedure to set the preview image? Also it would be interesting if i can set the preview image as an image I provided separately.

**Comment by johnshaughnessy** [comment, marked as answer]
> I thought spoke was using the current cam as preview image for the scene, but now I tried and was not able to set it up.

As of [this recent change to Spoke](https://github.com/mozilla/Spoke/pull/1011), the preferred way to set up the scene image is to use the Scene Preview Camera.

![image](https://user-images.githubusercontent.com/4072106/93005785-38856b00-f509-11ea-9d42-1ab7f1396be4.png)

This is an object that will live in the scene hierarchy and can be positioned using the same gizmos as any other object. There is also a button to set it based on your current viewport:

![image](https://user-images.githubusercontent.com/4072106/93005810-a16ce300-f509-11ea-8760-784144a24b62.png)

![image](https://user-images.githubusercontent.com/4072106/93005821-c4979280-f509-11ea-9be6-490171a313db.png)

The benefit of this is that you can configure the camera once, and don't have to adjust the viewport each time you want to re-upload the scene.

> Also it would be interesting if i can set the preview image as an image I provided separately.

As far as I know, that is not possible yet, but sounds like a good feature request. I recommend [looking for or adding a new issue](https://github.com/mozilla/spoke/issues) about this.

**Comment by salahzar** [comment]
thanks @johnshaughnessy this solves my question  at least partially... but... I can see the preview cam in the hierarchy but CANT see it as elements: is that something wanted? 
UPDATE: i've seen that if you deleted the cam in the hierarchy then the element appears in the selectable list so definitely this is solved 👍 

![image](https://user-images.githubusercontent.com/138039/93011412-66fd5780-f596-11ea-8ddc-657122f40422.png)
